langchain/tools/base.py

```diff
@@ -60,15 +69,16 @@ def _parse_input(
             if input_args is not None:
                 input_args.validate(tool_input)
 
-    @validator("callback_manager", pre=True, always=True)
-    def set_callback_manager(
-        cls, callback_manager: Optional[BaseCallbackManager]
-    ) -> BaseCallbackManager:
-        """If callback manager is None, set it.
-
-        This allows users to pass in None as callback manager, which is a nice UX.
-        """
-        return callback_manager or get_callback_manager()
+    @root_validator()
+    def raise_deprecation(cls, values: Dict) -> Dict:
+        """Raise deprecation warning if callback_manager is used."""
+        if "callback_manager" in values:
+            warnings.warn(
+                "callback_manager is deprecated. Please use callbacks instead.",
+                DeprecationWarning,
+            )
+        values["callbacks"] = values.pop("callback_manager", None)
```

Comment:
This seems to prefer the deprecated arg to the new arg, which is a little odd. Can be fixed by indenting this line into the if statement